static/app/views/replays/detail/console/index.tsx

```diff
@@ -47,12 +52,30 @@ function Console({breadcrumbs, startTimestampMs}: Props) {
     [breadcrumbs]
   );
 
+  const deps = useMemo(() => [items], [items]);
   const {cache, updateList} = useVirtualizedList({
     cellMeasurer,
     ref: listRef,
-    deps: [items],
+    deps,
   });
 
+  const handleDimensionChange = useCallback(
+    (index: number, path: string, expandedState: Record<string, boolean>) => {
+      const rowState = expandPaths.current.get(index) || new Set();
+      if (expandedState[path]) {
+        rowState.add(path);
+      } else {
+        // Collapsed, i.e. its default state, so no need to store state
+        rowState.delete(path);
+      }
+      expandPaths.current.set(index, rowState);
+      cache.clear(index, 0);
+      listRef.current?.recomputeGridSize({rowIndex: index});
+      listRef.current?.forceUpdateGrid();
```

Comment:
If we change the expanded state, clear out the cache for that specific row and recompute size and force re-render.